Samples/MediaPlayerCS/MainPage.xaml.cs

```diff
@@ -60,6 +60,14 @@ public VideoEffectConfiguration VideoEffectConfiguration
         public MainPage()
         {
             Config = new FFmpegInteropConfig();
+            Config.VideoOutputAllowBgra8 = true;
```

Comment:
Need to undo hard coded effects in sample. Maybe extend sample to insert FFmpeg video effects through button press.

Reply:
Yes. I agree. That was a lazy way of setting the filters from my end.

I have a few very busy weeks. I'll see what I can do about it.

Reply:
Done